### PR TITLE
add note about private channel behavior

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4491

### CODE REVIEW

**Comment by LolaJacobsen** [comment]
Getting review from PM Suparno Chakrabarti.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -134,6 +134,9 @@ Teams presence in Outlook is supported on the Outlook 2013 desktop app and later
 
 Each team in Microsoft Teams has a team site in SharePoint Online, and each channel in a team gets a folder within the default team site document library. Files shared within a conversation are automatically added to the document library, and permissions and file security options set in SharePoint are automatically reflected within Teams.
 
+> [!NOTE]
+> Each [private channel](https://docs.microsoft.com/en-us/microsoftteams/private-channels) has its own SharePoint site collection optimized for file sharing and fast provisioning
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:138]:
```suggestion
> Each [private channel](https://docs.microsoft.com/microsoftteams/private-channels) has its own SharePoint site collection optimized for file sharing and fast provisioning
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
Checking with PM Roshin Ramesan on this.

**Comment by LolaJacobsen** [comment]
Ping'd PM Arun Das again.

**Pull request merged**